shapely/tests/test_io.py

```diff
@@ -751,6 +1085,39 @@ def test_pickle_z(geom):
         assert actual.has_z
 
 
+@pytest.mark.skipif(
+    shapely.geos_version < (3, 12, 0),
+    reason="M coordinates not supported with GEOS < 3.12",
+)
+@pytest.mark.parametrize("geom", all_types_m)
+def test_pickle_m(geom):
+    if shapely.get_type_id(geom) == shapely.GeometryType.LINEARRING:
+        pytest.xfail("TODO: M gets dropped for some reason")
```

Comment:
Do you want to address that here, or open a follow-up issue? (a follow-up is fine for me as well , but we should track it to ensure it ideally gets done for 2.1)

Reply:
Tracked with #2005, and comment adjusted for this PR